### DISCUSSION TITLE
Click beatmap wedge: view beatmap background art and fullscreen music player

### DISCUSSION THREAD

**Discussion by yuuiko** [ppy/osu#13031] [Ideas]
I want to fly into that wedge and admire the art!
<img width="562" alt="Screen Shot 2019-06-03 at 9 42 21 pm" src="https://user-images.githubusercontent.com/50823728/58799365-7d54fd80-8648-11e9-9085-8f89cd2e6a17.png">

Additionally while viewing, maybe display an artist name and artwork weblink in a small label at the bottom corner, with a save to files button. These will only be visible if the mouse is moving.

Click anywhere except the save button to exit to song select.

If no metadata provided for the art, display nothing but the download button and gradient.

Designs:
![artview](https://user-images.githubusercontent.com/50823728/58801311-809eb800-864d-11e9-81a1-1674297ab84d.png)

![artview mask](https://user-images.githubusercontent.com/50823728/58801498-07ec2b80-864e-11e9-9fb5-c840f4fb9264.png)
Gradient: #000, at 50% opacity.
Text: White with the usual drop shadow, but 75% opacity.
Ignore the grey border around that image, it was just to differentiate it from the white github.

**Comment by peppy** [comment]
I'm not sure this is in scope. Also currently we do not have any embedded metadata for artwork creators.

A better option here may be improving the cinema mod experience.

**Comment by yuuiko** [comment]
Sure, focus on the scope first. 

My opinion: (Take it or leave it, you do have a lot to do)

Flying into the beatmap wedge is much easier than heading into cinema and disabling toggles. One can click it, have a peep and fly back out. Plus, clicking it doesn’t already do anything so why not really?

I’m aware there is no artwork metadata, it would be great if this was introduced because I feel the artwork is, well, art and should be appreciated. In my design spec however I note how no metadata would be handled.

Second, I think this should be some kind of music player screensaver. When the current song is finished it’ll fade to the next background and keep playing. In this case maybe it’ll be good to add the song name to the right and playback controls in the centre when the mouse is moving.
